src/Features/VisualBasic/Portable/UseObjectInitializer/VisualBasicUseObjectInitializerCodeFixProvider.vb

```diff
@@ -67,7 +67,7 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.UseObjectInitializer
                 Dim initializer = SyntaxFactory.NamedFieldInitializer(
                     keyKeyword:=Nothing,
                     dotToken:=match.MemberAccessExpression.OperatorToken,
-                    name:=DirectCast(match.MemberAccessExpression.Name, IdentifierNameSyntax),
+                    name:=SyntaxFactory.IdentifierName(match.MemberName),
```

Comment:
this likely will not work for names that are keywords.  You store .ValueText which won't have hte propery escaping here.

Reply:
@CyrusNajmabadi I've tested it with ```Class```, ```Property``` and ```Date``` keywords. It works, probably we don't need to escape names in object initializer.